ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/ovn/ipsec/provision/cucushift-installer-rehearse-aws-ipi-private-ovn-ipsec-provision-chain.yaml

```diff
@@ -13,7 +13,7 @@ chain:
     - ref: ovn-conf
     - ref: ovn-conf-ipsec-manifest
     - ref: ipi-install-install
-    - ref: mirror-images-enable-qe-catalogsource
+    - ref: enable-qe-catalogsource-disconnected
```

Comment:
This one should use `enable-qe-catalogsource`

Reply:
Thanks! Updated it.